src/content/docs/query-your-data/explore-query-data/use-charts/chart-types.mdx

```diff
@@ -57,669 +57,623 @@ This table contains all chart types. Whether a chart type is available for your
-    </figcaption>
-
-    Use histograms to show the distribution of data across time buckets. See [Histograms: View data distribution](/docs/using-new-relic/user-interface-functions/view-your-data/histograms-view-data-distribution) for more detailed information about histogram-based charts.
-
-    <table>
-      <tbody>
-        <tr>
-          <td>
-            Use a histogram when you want to
-          </td>
-
-          <td>
-            Chart frequency of data over a series of values.
-          </td>
-        </tr>
-
-        <tr>
-          <td>
-            Avoid a histogram when you want to
-          </td>
-
-          <td>
-            View the same data as [heat](#widget-heatmap) [map](#widget-heatmap) or to view the data without the time buckets.
-          </td>
-        </tr>
-      </tbody>
-    </table>
-  </Collapser>
-
-  <Collapser
-    className="freq-link"
-    id="widget-json"
-    title="JSON"
-  >
-    Use the returned JSON to review the raw data returned and to help develop integrations with New Relic. To implement integrations, use [remote queries](/docs/insights/new-relic-insights/adding-querying-data/querying-your-data-remotely) to return raw JSON as a service.
-
-    To use this, select the <DoNotTranslate>**Share**</DoNotTranslate> menu and click <DoNotTranslate>**Copy JSON**</DoNotTranslate>.
-
-    Example NRQL query:
-
-    ```
-    SELECT uniqueCount(name) FROM Transaction
-    WHERE http.statusCode = '200'
-    TIMESERIES
-    FACET appName
-    LIMIT 10
-    SINCE 24 hours ago
-    ```
-
-    Example JSON response:
-
-    ```
-    {
-      "metadata": {
-        "contents": {
-          "messages": [],
-          "timeSeries": {
-            "messages": [],
-            "contents": [
-              {
-                "function": "uniquecount",
-                "attribute": "name",
-                "simple": true
-              }
-            ]
-          },
-          "bucketSizeMillis": 1800000
-        },
-        "eventTypes": [
-          "Transaction"
-        ],
-        "eventType": "Transaction",
-        "openEnded": true,
+To enable the data for a billboard chart, search for a single value. You can view multiple values by adding a facet from the dropdown.
+
+<DoNotTranslate>**Thresholds**</DoNotTranslate>
+
+Optionally, you can set <DoNotTranslate>**Warning**</DoNotTranslate> and/or <DoNotTranslate>**Critical**</DoNotTranslate> thresholds on billboards. Once set, if your service is working correctly, the chart's full box is green. When a threshold is crossed, the chart turns yellow (warning) or red (critical). Access the threshold fields on the right side of the chart edit pane in the query builder.
+
+<img
+  title="A billboard chart."
+  alt="A screenshot of a billboard chart."
+  src={queriesnrqlBillboardsColor}
+/>
+
+<figcaption>
+  The billboard chart displays a single value in a large, easy to read font. The background color changes based on your thresholds.
```

Comment:
So I may be completely off here, but I think that statements like "easy to read" (or "easy to do" or the like) are something that we typically avoid because it's too nebulous of a claim?

Reply:
It's pretty non-speciifc, isn't it? Changed to 

>The billboard chart displays a single value in an extra-large font, which can help draw attention to a specific value in a busy dashboard.